cli/src/cmd/forge/inspect.rs

```diff
@@ -164,6 +164,23 @@ impl Cmd for InspectArgs {
                     ))?
                 );
             }
+            ContractArtifactField::Errors => {
+                let mut out = serde_json::Map::new();
+                if let Some(LosslessAbi { abi, .. }) = &artifact.abi {
+                    // Print the signature of all errors
+                    for er in abi.errors.iter().flat_map(|(_, errors)| errors) {
+                        let types =
+                            er.inputs.iter().map(|p| p.kind.to_string()).collect::<Vec<_>>();
+                        let sig = format!("{:x}", er.signature());
+                        let sig_trimmed = &sig[0..8];
+                        out.insert(
+                            format!("{}({})", er.name, types.join(",")),
+                            sig_trimmed.to_string().into(),
+                        );
```

Comment:
this seems fine,
no strong opinion on the format here tbh,

defer to @Evalir

Reply:
I like this! Tested and format is the same as when printing events—looks good IMHO.